src/BootstrapTable.js

```diff
@@ -142,7 +142,13 @@ class BootstrapTable extends Component {
     const { options, selectRow } = nextProps;
 
     this.store.setData(nextProps.data.slice());
-    let page = options.page || this.state.currPage;
+
+    let page;
+    if (options.page !== 'undefined') {
```

Comment:
yeah thats how i originally did it, eslint was yelling at me so i thought this may have been a pattern in the code base. if @AllenFang agrees we can just adjust the `.eslintrc` to allow for double equal null checks.